c/Systems_OpenBSD_MemSafety.cfg

```diff
@@ -0,0 +1,2 @@
+Description: If the EtherIP tunnel protocol was disabled, IPv6 packets were not discarded properly. This causes a double free in the kernel.
```

Comment:
This should describe the corresponding category, take for example https://github.com/sosy-lab/sv-benchmarks/blob/master/c/Systems_DeviceDriversLinux64_ReachSafety.cfg